src/htmltojsx.js

```diff
@@ -562,12 +579,9 @@ StyleParser.prototype = {
    */
   toJSXString: function() {
     var output = [];
-    for (var key in this.styles) {
-      if (!this.styles.hasOwnProperty(key)) {
-        continue;
-      }
-      output.push(this.toJSXKey(key) + ': ' + this.toJSXValue(this.styles[key]));
-    }
+    eachObj(this.styles, function(key, value) {
```

Comment:
We have refactored the iteration of the object using the newly created `eachObj` helper.
I'm sorry I made a confusing change... :cry: